guide/preparations/adding-your-bot-to-servers.md

```diff
@@ -9,28 +9,35 @@ Before you're able to see your bot in your own (or other) servers, you'll need t
 The basic version of one such link looks like this:
 
 ```
-https://discord.com/oauth2/authorize?client_id=123456789012345678&scope=bot+applications.commands
+https://discord.com/api/oauth2/authorize?client_id=123456789012345678&permissions=8&scope=bot%20applications.commands
```

Comment:
```suggestion
https://discord.com/api/oauth2/authorize?client_id=123456789012345678&permissions=0&scope=bot%20applications.commands
```
It's recommended to select the specific permissions you need instead of choosing `Administrator`, unless you know why you need it. It shouldn't be used in the guide